Havit.Blazor.Components.Web.Bootstrap/Forms/HxInputNumber.cs

```diff
@@ -90,7 +90,7 @@ public class HxInputNumber<TValue> : HxInputBaseWithInputGroups<TValue>, IInputW
 	InputSize IInputWithSize.InputSizeEffective => InputSizeEffective;
 
 	/// <summary>
-	/// Gets or sets the behavior when the model is updated from the input.
```

Comment:
@hakenr the comment has been taken from HxInputText so if you don't like it, it should be adjusted there as well.